src/createBuildConfigs.ts

```diff
@@ -15,7 +15,8 @@ let tsdxConfig = {
 };
 
 if (fs.existsSync(paths.appConfig)) {
-  tsdxConfig = require(paths.appConfig);
+  const customConfig = require(paths.appConfig);
+  tsdxConfig = { ...tsdxConfig, ...customConfig };
```

Comment:
why was this changed? it doesn't seem to have been in your original commit either 🤔

Reply:
without this patch tsdx will fall there https://github.com/jaredpalmer/tsdx/blob/216bb56392adf066ea959577701314d0ca899b16/src/createBuildConfigs.ts#L41 in case `tsdx.config.js` will not have { rollup } section

Reply:
but it wasn't failing before??? I'm very confused, it started failing here right?

Reply:
yes! there are no tests with fixture `build-withConfig`
proof:
![Screenshot from 2020-03-11 09-31-04](https://user-images.githubusercontent.com/380645/76393073-77c8d880-637b-11ea-878c-df67e2a01186.png)
![Screenshot from 2020-03-11 09-29-51](https://user-images.githubusercontent.com/380645/76393080-7ac3c900-637b-11ea-879d-60214be6fa53.png)

Reply:
https://github.com/jaredpalmer/tsdx/issues/572

Reply:
> yes! there are no tests with fixture `build-withConfig`

Yes, I know that -- I mentioned it's not set-up for automated testing yet and that needs to be fixed at some point.

> #572

See my response there. This change makes sense when adding `closureCompilerOptions` to `tsdx.config.js`, but I don't think we should do that at this time.

Reply:
please, check updated tests

Reply:
removed